daprdocs/content/en/developing-applications/building-blocks/bindings/howto-triggers.md

```diff
@@ -18,79 +18,265 @@ Dapr bindings allow you to:
 
 For more info on bindings, read [this overview]({{<ref bindings-overview.md>}}).
 
-For a quickstart sample showing bindings, visit this [link](https://github.com/dapr/quickstarts/tree/master/bindings).
+## Example:
+
+The below code example loosely describes an application that processes orders. In the example, there is an order processing service which has a Dapr sidecar. The checkout service uses Dapr to trigger the application(input binding).
+
+<img src="/images/building-block-bindings-example.png" width=1000 alt="Diagram showing bindings of example service">
```

Comment:
this image has an extra thin grey line at the right hand side that needs to be cropped out.